dpdata/plugins/ase.py

```diff
@@ -187,6 +188,135 @@ def to_labeled_system(self, data, *args, **kwargs):
         return structures
 
 
+@Format.register("ase/traj")
+class ASETrajFormat(Format):
```

Comment:
A unit test should be added for this test. I found there have already been testing cases in `tests/test_ase_traj.py` and related trajectory files, so just writing in that file is okay.

Reply:
I am not familiar with unit test, however I tried to put a simple test in that file.
Please check it